File: spring/src/main/java/io/swagger/model/MainCollection.java
package io.swagger.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import org.springframework.validation.annotation.Validated;
import javax.validation.Valid;
import javax.validation.constraints.*;

/**
 * MainCollection
 */
@Validated
@javax.annotation.Generated(value = "io.swagger.codegen.languages.SpringCodegen", date = "2022-10-07T23:22:34.385Z")


public class MainCollection   {
  @JsonProperty("cid")
  private String cid = null;

  @JsonProperty("createdAt")
  private String createdAt = null;

  @JsonProperty("description")
  private String description = null;

  @JsonProperty("name")
  private String name = null;

  @JsonProperty("userId")
  private Integer userId = null;

  @JsonProperty("uuid")
  private String uuid = null;

  public MainCollection cid(String cid) {
    this.cid = cid;
    return this;
  }

  /**
   * Get cid
   * @return cid
  **/
  @ApiModelProperty(value = "")


  public String getCid() {
    return cid;
  }

  public void setCid(String cid) {
    this.cid = cid;
  }

  public MainCollection createdAt(String createdAt) {
    this.createdAt = createdAt;
    return this;
  }

  /**
   * Get createdAt
   * @return createdAt
  **/
  @ApiModelProperty(value = "")


  public String getCreatedAt() {
    return createdAt;
  }

  public void setCreatedAt(String createdAt) {
    this.createdAt = createdAt;
  }

  public MainCollection description(String description) {
    this.description = description;
    return this;
  }

  /**
   * Get description
   * @return description
  **/
  @ApiModelProperty(value = "")


  public String getDescription() {
    return description;
  }

  public void setDescription(String description) {
    this.description = description;
  }

  public MainCollection name(String name) {
    this.name = name;
    return this;
  }

  /**
   * Get name
   * @return name
  **/
  @ApiModelProperty(value = "")


  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public MainCollection userId(Integer userId) {
    this.userId = userId;
    return this;
  }

  /**
   * Get userId
   * @return userId
  **/
  @ApiModelProperty(value = "")


  public Integer getUserId() {
    return userId;
  }

  public void setUserId(Integer userId) {
    this.userId = userId;
  }

  public MainCollection uuid(String uuid) {
    this.uuid = uuid;
    return this;
  }

  /**
   * Get uuid
   * @return uuid
  **/
  @ApiModelProperty(value = "")


  public String getUuid() {
    return uuid;
  }

  public void setUuid(String uuid) {
    this.uuid = uuid;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    MainCollection mainCollection = (MainCollection) o;
    return Objects.equals(this.cid, mainCollection.cid) &&
        Objects.equals(this.createdAt, mainCollection.createdAt) &&
        Objects.equals(this.description, mainCollection.description) &&
        Objects.equals(this.name, mainCollection.name) &&
        Objects.equals(this.userId, mainCollection.userId) &&
        Objects.equals(this.uuid, mainCollection.uuid);
  }

  @Override
  public int hashCode() {
    return Objects.hash(cid, createdAt, description, name, userId, uuid);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class MainCollection {\n");
    
    sb.append("    cid: ").append(toIndentedString(cid)).append("\n");
    sb.append("    createdAt: ").append(toIndentedString(createdAt)).append("\n");
    sb.append("    description: ").append(toIndentedString(description)).append("\n");
    sb.append("    name: ").append(toIndentedString(name)).append("\n");
    sb.append("    userId: ").append(toIndentedString(userId)).append("\n");
    sb.append("    uuid: ").append(toIndentedString(uuid)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: dart/lib/model/main_collection.dart
part of swagger.api;

class MainCollection {
  
  String cid = null;
  

  String createdAt = null;
  

  String description = null;
  

  String name = null;
  

  int userId = null;
  

  String uuid = null;
  
  MainCollection();

  @override
  String toString() {
    return 'MainCollection[cid=$cid, createdAt=$createdAt, description=$description, name=$name, userId=$userId, uuid=$uuid, ]';
  }

  MainCollection.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    cid =
        json['cid']
    ;
    createdAt =
        json['createdAt']
    ;
    description =
        json['description']
    ;
    name =
        json['name']
    ;
    userId =
        json['userId']
    ;
    uuid =
        json['uuid']
    ;
  }

  Map<String, dynamic> toJson() {
    return {
      'cid': cid,
      'createdAt': createdAt,
      'description': description,
      'name': name,
      'userId': userId,
      'uuid': uuid
     };
  }

  static List<MainCollection> listFromJson(List<dynamic> json) {
    return json == null ? new List<MainCollection>() : json.map((value) => new MainCollection.fromJson(value)).toList();
  }

  static Map<String, MainCollection> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, MainCollection>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new MainCollection.fromJson(value));
    }
    return map;
  }
}

